packages/components/src/button/src/Button.css

```diff
@@ -78,159 +78,213 @@ a.o-ui-button {
     min-height: var(--o-ui-button-height-sm);
 }
 
+.o-ui-button-sm .o-ui-button-text {
+    font-size: var(--hop-body-sm-medium-font-size);
+}
+
 .o-ui-button-md {
     min-height: var(--o-ui-button-height-md);
 }
 
+.o-ui-button-md .o-ui-button-text {
+    font-size: var(--hop-body-md-medium-font-size);
+}
+
 /* PRIMARY */
 .o-ui-button-primary {
-    color: var(--o-ui-text-alias-static-white);
-    background-color: var(--o-ui-bg-alias-accent);
+    background-color: var(--hop-primary-surface-strong);
+    color: var(--hop-primary-text-strong);
 }
 
 /* PRIMARY | HOVER */
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading):hover,
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading).o-ui-button-hover {
-    background-color: var(--o-ui-bg-alias-accent-hover);
+    background-color: var(--hop-primary-surface-strong-hover);
 }
 
 /* PRIMARY | HOVER | ACTIVE */
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading):hover:active,
```

Comment:
Something I ran into on Igloo, it seems that they didn't want the active style when you are clicking the button:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/87b2f848-f3c0-4b94-9a32-ddab0eccaf95)

It has the active style when a button is selected. So in this example, I'm guessing no style for :active and only for .-o-ui-button-active.
Maybe this is not the case for Orbit. I'd validate with design.

Reply:
Plus, it is pretty annoying click buttons and sometimes it goes from really dark and back to light when you let go.

Reply:
Indeed a discussion to have. Although this behavior is the standard behavior for a button. Orbit aims to be accessible and follow web standards. Having no active states seems against best practices.

The -active is used to test active states and is not used in production only for tests.

Reply:
https://usabilitypost.com/2008/12/16/pressed-button-state-with-css/

Reply:
Upon talking with designers, we might add an active(pressed) state in Hopper, for now I'll follow what they designed.